homeassistant/components/sabnzbd/__init__.py

```diff
@@ -147,88 +39,50 @@ class SabnzbdSensorEntityDescription(SensorEntityDescription, SabnzbdRequiredKey
     {
         DOMAIN: vol.Schema(
             {
-                vol.Required(CONF_API_KEY): cv.string,
-                vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
-                vol.Optional(CONF_PATH): cv.string,
-                vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-                vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
-                vol.Optional(CONF_SENSORS): vol.All(
-                    cv.ensure_list, [vol.In(SENSOR_KEYS)]
-                ),
-                vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
+                vol.Required(CONF_API_KEY): str,
+                vol.Optional(CONF_NAME, default=DEFAULT_NAME): str,
+                vol.Required(CONF_URL): str,
+                vol.Optional(CONF_PATH): str,
```

Comment:
We should not remove the legacy options completely from the config schema, but mark them as deprecated instead with `cv.deprecated`. This will avoid a breaking change for users.